test/powershell/Modules/Microsoft.PowerShell.Utility/Select-Object.Tests.ps1

```diff
@@ -225,7 +225,7 @@ Describe "Select-Object DRT basic functionality" -Tags "CI" {
 		$results.Count | Should -Be 0
 	}
 
-	It "Select-Object with Start-Time In Idle Process should work"{
+	It "Select-Object with Start-Time In Idle Process should work" -Skip:(!$IsWindows) {
 		$results = Get-Process i* | Select-Object ProcessName
```

Comment:
Why not just change it to `Get-Process *`  removing the `i` from `i*` ?

Reply:
Fixed.